cms/search/utils.py

```diff
@@ -0,0 +1,69 @@
+from typing import TYPE_CHECKING
+
+from django.apps import apps
+from django.utils.encoding import force_str
+from wagtail.rich_text import get_text_for_indexing
+
+from cms.release_calendar.enums import ReleaseStatus
+
+if TYPE_CHECKING:
+    from wagtail.models import Page
+
+
+def build_standard_resource_dict(page: "Page") -> dict:
+    """Returns a dict with the standard resource fields.
+    This covers the non-release case (and also forms the base of the release case).
+    """
+    return {
+        "uri": page.url_path,
```

Comment:
This currently contains the full path which includes the `home`path; we need to omit that. So instead of `"/home/releases/jan-2025/"` it should be `"/releases/jan-2025/"`. See: https://github.com/ONSdigital/dis-search-upstream-stub/blob/main/data/json_files/release_two_topics.json

Reply:
Good spot! Done!